src/main/java/frc/trigon/robot/subsystems/arm/ArmConstants.java

```diff
@@ -0,0 +1,177 @@
+    private static final CANSparkMax.IdleMode
+            ANGLE_IDLE_MODE = CANSparkMax.IdleMode.kBrake,
+            ELEVATOR_IDLE_MODE = CANSparkMax.IdleMode.kBrake;
+    private static final boolean
+            MASTER_ANGLE_INVERTED = false,
+            FOLLOWER_ANGLE_INVERTED = false,
+            MASTER_ELEVATOR_INVERTED = false,
+            FOLLOWER_ELEVATOR_INVERTED = false,
+            ELEVATOR_ENCODER_PHASE = false;
+    private static final SensorDirectionValue ANGLE_ENCODER_SENSOR_DIRECTION_VALUE = SensorDirectionValue.CounterClockwise_Positive;
+    private static final double VOLTAGE_COMPENSATION_SATURATION = 12;
+    private static final double
+            ANGLE_ENCODER_OFFSET = 0,
+            ELEVATOR_ENCODER_OFFSET = 0;
+    private static final AbsoluteSensorRangeValue ANGLE_ENCODER_RANGE_VALUE = AbsoluteSensorRangeValue.Signed_PlusMinusHalf;
+    static final CANSparkMax
+            MASTER_ELEVATOR_MOTOR = new CANSparkMax(MASTER_ELEVATOR_MOTOR_ID, CANSparkMaxLowLevel.MotorType.kBrushless),
+            FOLLOWER_ELEVATOR_MOTOR = new CANSparkMax(FOLLOWER_ANGLE_MOTOR_ID, CANSparkMaxLowLevel.MotorType.kBrushless),
+            MASTER_ANGLE_MOTOR = new CANSparkMax(MASTER_ANGLE_MOTOR_ID, CANSparkMaxLowLevel.MotorType.kBrushless),
+            FOLLOWER_ANGLE_MOTOR = new CANSparkMax(FOLLOWER_ELEVATOR_MOTOR_ID, CANSparkMaxLowLevel.MotorType.kBrushless);
+    static final CANcoder ANGLE_ENCODER = new CANcoder(ANGLE_ENCODER_ID);
+    static final WPI_TalonSRX ELEVATOR_ENCODER = new WPI_TalonSRX(ELEVATOR_ENCODER_ID);
+    static final double ELEVATOR_METERS_PER_REVOLUTION = 1;
+    static final double
+            ANGLE_TOLERANCE_DEGREES = 1,
+            ELEVATOR_TOLERANCE_DEGREES = 1;
+
+    private static final double
+            MAX_ELEVATOR_VELOCITY = 0,
+            MAX_ELEVATOR_ACCELERATION = 0,
+            MAX_ANGLE_VELOCITY = 0,
+            MAX_ANGLE_ACCELERATION = 0;
+    static final TrapezoidProfile.Constraints ELEVATOR_CONSTRAINTS = new TrapezoidProfile.Constraints(
+            MAX_ELEVATOR_VELOCITY,
+            MAX_ELEVATOR_ACCELERATION
+    ),
+            ANGLE_CONSTRAINTS = new TrapezoidProfile.Constraints(
+                    MAX_ANGLE_VELOCITY,
+                    MAX_ANGLE_ACCELERATION
+            );
+
+    static final StatusSignal<Double>
+            ANGLE_ENCODER_POSITION_SIGNAL = ANGLE_ENCODER.getPosition(),
+            ANGLE_ENCODER_VELOCITY_SIGNAL = ANGLE_ENCODER.getVelocity();
+
+    private static final double
+            ANGLE_P = 0,
+            ANGLE_I = 0,
+            ANGLE_D = 0,
+            ELEVATOR_P = 0,
+            ELEVATOR_I = 0,
+            ELEVATOR_D = 0;
+
+    static final PIDController
+            ANGLE_PID_CONTROLLER = new PIDController(
+            ANGLE_P,
+            ANGLE_I,
+            ANGLE_D
+    ),
+            ELEVATOR_PID_CONTROLLER = new PIDController(
+            ELEVATOR_P,
+            ELEVATOR_I,
+            ELEVATOR_D
+    );
```

Comment:
Reformat class and download the save actions - xdev edition and configure it as I mentioned in the group.